crates/sui-framework/sources/governance/validator.move

```diff
@@ -33,9 +33,9 @@ module sui::validator {
         /// The network address of the validator (could also contain extra info such as port, DNS and etc.).
         net_address: vector<u8>,
         /// Total amount of validator stake that would be active in the next epoch.
-        /// This only includes validator stake, and does not include delegation.
-        /// TODO: stake should include delegated stake: https://github.com/MystenLabs/sui/issues/2834
         next_epoch_stake: u64,
+        /// Total amount of delegated stake that would be active in the next epoch.
+        next_epoch_delegation: u64,
```

Comment:
Similar question: do you think we could merge this into next_epoch_stake?

Reply:
Yes we could do merge these two if stake always includes both self stake and delegation. But what if later we want to do something different and want to have them separate? It might be better to keep the granularity especially in the on-chain data. 
I don't feel too strongly about it though. I'm okay with merging them.

Reply:
That's fair. We can leave them separated and see.